docs/docs/faq/question_05.md

```diff
@@ -36,4 +36,26 @@ with open("/path/to/my_inference.pkl", "wb") as handle:
 ```
 However, saving and loading the `inference` object will slightly modify the object (in order to make it serializable). These modifications lead to the following two changes in behaviour:
 1) Retraining from scratch is not supported, i.e. `.train(..., retrain_from_scratch=True)` does not work.
-2) When the loaded object calls the `.train()` method, it generates a new tensorboard summary writer (instead of appending to the current one).
\ No newline at end of file
+2) When the loaded object calls the `.train()` method, it generates a new tensorboard summary writer (instead of appending to the current one).
+
+## I trained a model on a GPU. Can I load it on a CPU?
+
+Yes. This can be useful for quick inspection and plotting. We advise not to continue training on a CPU.
```

Comment:
Do we advise this or does it not work?

Reply:
You can hack it by changing the device of the inference object (`inference._device = "cpu"`), and then continuing to train as normal. I think we would want to then add a `.to(device)` method for `NeuralInference` if we want to provide this as an option, but I am not sure we want to support this.

Reply:
Please make this explicit in the FAQ.